src/Builder/FrontendDialectTransformer.cpp

```diff
@@ -1065,9 +1065,7 @@ class FrontendGenImpl {
       if (v.empty()) {
         // Missing (optional) parameter.
         operandOnnxTypes.push_back(unspecifiedType);
-        auto no_value = builder_.create<ONNXNoneOp>(UnknownLoc());
-
-        operands.push_back(no_value);
+        operands.push_back(none());
```

Comment:
Where does the `none()` definition comes from? Since `none` is such a generic term, and if a none value is created here, maybe a `createNoneValue()` might be a bit more explicit

Reply:
good idea, I renamed it as you suggested

it's method in the enclosing class FrontendGenImpl